huaweicloud/services/waf/data_source_huaweicloud_waf_instance_groups.go

```diff
@@ -110,6 +110,12 @@ func DataSourceWafInstanceGroups() *schema.Resource {
 								},
 							},
 						},
+						"load_balances": {
+							Type:       schema.TypeList,
+							Computed:   true,
```

Comment:
this is attribute as well.

Reply:
considering that this argument may be used for data integration or reference in Terraform scripts, so it is reserved